packages/twenty-front/src/generated/graphql.ts

```diff
@@ -2310,6 +2334,11 @@ export type MutationValidateApprovedAccessDomainArgs = {
 };
 
 
+export type MutationVerifyOutboundMessageDomainArgs = {
+  input: VerifyOutboundMessageDomainInput;
```

Comment:
Bad pattern frequently generated by AI. Let's put the input directly here, no need to wrap it in an input property